CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## v0.7.0
+
+* Experiment can now specify multiple qualify blocks
+* Allow pass of an argument to qualify with a method name as a symbol, instead of a block
```

Comment:
could you explain that a breaking change was introduced?
`Verdict::Experiment#qualifier` has been removed and now there's `Verdict::Experiment#qualifiers`, which returns an array of procs.